takwimu/takwimu_ui/src/components/AnalysisContent/OtherInfoNav.js

```diff
@@ -29,9 +29,20 @@ const styles = theme => ({
     textDecoration: 'underline'
   },
   label: {
+    color: 'white',
     fontSize: '0.813rem',
     marginTop: '1rem',
     marginBottom: '0.5rem'
+  },
+  navigation: {
+    backgroundColor: 'unset',
+    padding: 0,
+    [theme.breakpoints.up('md')]: {
+      width: '100%'
+    },
+    [theme.breakpoints.up('lg')]: {
+      width: '100%'
```

Comment:
NP: Unnecessary, will use 'md' value.

Reply:
Necessarily to override ContentSection root.